src/pkgdev/scripts/pkgdev_mask.py

```diff
@@ -165,9 +168,12 @@ def __str__(self):
         return ''.join(self.header) + '\n\n'.join(map(str, self.masks))
 
 
-def get_comment():
+def get_comment(bugs):
     """Spawn editor to get mask comment."""
     tmp = tempfile.NamedTemporaryFile(mode='w')
+    if len(bugs) > 0:
+        # Bug #A, #B, #C
+        tmp.write(f'\nBug {", ".join([f"#{bug}" for bug in bugs])}')
```

Comment:
Using a list comprehension inside of .join() is unnecessary as it should work with the raw generator expression.

Reply:
Clearly your Python skills are better than mine :-)

Should I just drop the surrounding square brackets here?

Reply:
Honestly I'd probably do something similar to the following:

```python
from snakeoil.strings import pluralism

bug_list = ", ".join(f'#{b}' for b in bugs)
s = pluralism(bugs)
tmp.write(f'\nBug{s} {bug_list}')
```

Reply:
There's mixed conventions, and I thought maybe just sticking with "Bug" (not pluralising it) would be fine. But I'll integrate this.

Reply:
Adding the pluralism part is up to you, I was just giving an example of what's done elsewhere especially throughout pkgcheck results output.

Reply:
Done.